### DISCUSSION TITLE
Open all links in mini-browser

### DISCUSSION THREAD

**Discussion by wassimkha** [eclipse-theia/theia#9896] [Q&A]
Hi,
I'm trying to create a feature that will open any links on Theia by using the `mini-browser`, although most of the links that I try to open are not showing. I believe this is due to the fact that we try to open them inside of an iframe (which is what the `mini-browser` uses) and it blocks most protected websites. Is there any way around that, so that we might be able to open any public link inside of theia?
Thank you

**Comment by vince-fugnitto** [comment]
@wassimkha I believe it is related to the same discussions had in https://github.com/eclipse-theia/theia/discussions/9162, there does not seem to be a reliable solution at the moment to allow it.
